src/main/java/org/apache/tomee/website/Downloads.java

```diff
@@ -149,7 +149,7 @@ private static void printRow(Download d) {
                 "|" + new SimpleDateFormat("d MMM yyyy").format(Date.from(LocalDateTime.parse(d.date, RFC_1123_DATE_TIME).toInstant(ZoneOffset.UTC))) +
                 "|" + d.size + " MB " +
                 "|" + d.format.toUpperCase() +
-                "| " + d.url + "[icon:download[] " + d.format.toUpperCase() + "] " + d.sha1 + "[icon:download[] SHA1] " + d.md5 + "[icon:download[] MD5]");
+                "| " + d.url + "[icon:download[] " + d.format.toUpperCase() + "] " + d.sha1 + "[icon:download[] SHA1] " + d.md5 + "[icon:download[] MD5] " + d.asc + "[icon:download[] PGP]");
```

Comment:
think it misses SHA512 ;) (should be computed on the fly using local artifact if present and if not downloading it in m2 - tomee has MavenResolver+HttpResolver which should enable to do that smoothly)

Reply:
This is true (also for SHA256). Seems the page wasn't generated via `Downloads.java` for a long time.

The output between `Downloads` and the currenct `download-ng.adoc` also differs:

- `https://repo.maven.apache.org/maven2/org/apache/tomee/apache-tomee/...` versus `https://www.apache.org/dyn/closer.cgi/...` on the downloads-ng web page.

Reply:
Yep sadly which means the content became no more validated and potentially erroneous since I assume nothing checks the links work. I would keep the central links for binary since they are more reliable than mirrors generally BTW.

Reply:
I think, `Downloads` generates a download page based on `repo.maven.apache.org`, which does not contain SHA256 or SHA512 files. Afaik, Maven is capable of generating SHA256 / SHA512 checksums during publishing.

In general, it seems, the URL pattern was switched from `repo.maven.apache.org` to the mirror links `https://www.apache.org/dyn/closer.cgi/...` with `7.0.5 +`. 

> Yep sadly which means the content became no more validated and potentially erroneous since I assume nothing checks the links work.

This is true. Afaik, there is no automated process of doing this atm.

> Yep sadly which means the content became no more validated and potentially erroneous since I assume nothing checks the links work. I would keep the central links for binary since they are more reliable than mirrors generally BTW.

There is also an open issue to reduce disk load on the ASF mirrors, which was pinged by INFRA recently: https://issues.apache.org/jira/browse/TOMEE-1096 

Might be worth bringing this to the **dev@** list for further discussions?

Maybe @jgallimore , @jeanouii or @cesarhernandezgt also have some thoughts on it?